content/release-10-18-0/digital-twin-manager-10-18-0-bundle/10-18-0-1.md

```diff
@@ -0,0 +1,355 @@
+<td>CTM-207</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Assets move feature is applied with required CSS changes.</td>
+<td>CTM-560</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Roles and permissions implementation in DTM.</td>
+<td>CTM-649</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+</tbody></table></div>
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Description</th>
+<th>
+Issue</th>
+<th>
+Build version</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>A progress bar is shown during the processing of the file for <b>Import asset model</b>, <b>Import asset properties</b>, and <b>Asset model samples</b>.</td>
+<td>CTM-582</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Fixed duplication of asset models when imported from the <b>Asset model samples</b>.</td>
+<td>CTM-578</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>The <b>Export</b> button is disabled when there are no selected properties to be exported.</td>
+<td>CTM-577</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Deselecting the child asset models after exporting the asset models is fixed. </td>
+<td>CTM-576</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Modified the success messages that are shown while creating, updating, and deleting the <b>Asset properties</b>.</td>
+<td>CTM-562</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>On clicking the <b>Reload</b> button in the <b>Asset tree</b> tab, the expanded asset hierarchy is completely closed now.</td>
+<td>CTM-558</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Fixed CSS issues in the <b>Asset Models</b> page.</td>
+<td>CTM-554</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Child asset models and asset properties are shown on expansion of the parent asset model in the <b>Export Asset Models</b> popup.</td>
+<td>CTM-551</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>The issues observed in the <b>New asset</b> page are fixed.</td>
+<td>CTM-544</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>The <b>Import asset properties</b> dialog box is shown on click of <b>Import asset properties</b>.</td>
+<td>CTM-543</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td><b>Subassets</b> page now shows valid message if there are no assets/devices exists.</td>
+<td>CTM-368</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Selection and pagination issues in <b>Assets</b> page is now handled.</td>
+<td>CTM-545</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td><b>Asset model samples</b> page is now added with loading symbol while loading the sample models.</td>
+<td>CTM-579</td>
+<td>1018.0.1.202307111702-7614b35</td>
+</tr>
+
+<tr>
+<td>Assets move popup is implemented with pagination and search functionality.</td>
```

Comment:
@spau6 Can you provide a screenshot of the dialog window?

Reply:
![image](https://github.com/SoftwareAG/c8y-releasenotes/assets/109800638/e44e3286-4e9a-4022-ab6d-19a8e43bf5ca)

Reply:
Please find the uploaded image

Reply:
```suggestion
<td>The <b>Move assets</b> dialog window is implemented with pagination and search functionality.</td>
```